helm/examples/azure/hpcc-azurefile/values.schema.json

```diff
@@ -71,6 +71,14 @@
         "shareName": {
           "description": "Azure storage share name",
           "type": "string"
+        },
+        "secretName": {
+          "description": "the secretName, the storage account, for current plane",
+          "type": "string"
+        },
+        "secretNamespace": {
+          "description": "the namespace of the secret, storage account",
```

Comment:
similarly:
"Kubernetes namespace for the Kubernetes secret named by secretName"

Reply:
Fixed